src/cli.rs

```diff
@@ -37,23 +37,29 @@ pub fn execute() -> Result<(), Error> {
     let matches = app.get_matches();
     let config_path = matches.value_of("config").unwrap_or(".relay");
 
-    // config init is special because it does not yet have a config.
+    // Commands that do not need to load the config:
     if let Some(matches) = matches.subcommand_matches("config") {
         if let Some(matches) = matches.subcommand_matches("init") {
             return init_config(&config_path, &matches);
         }
-    // likewise completions generation does not need the config.
     } else if let Some(matches) = matches.subcommand_matches("generate-completions") {
         return generate_completions(&matches);
-    // we also do not read the config for offline event processing
     } else if let Some(matches) = matches.subcommand_matches("process-event") {
         return process_event(&matches);
+    } else if let Some(_matches) = matches.subcommand_matches("event-json-schema") {
+        #[cfg(feature = "jsonschema")]
+        return event_json_schema(&_matches);
```

Comment:
This doesn't have to be acted on now but rather in a follow-up: Let's move this into a separate binary that depends on `relay-general` instead of a command on the main Relay binary.